composer.json

```
@@ -3,7 +3,7 @@
   "type": "library",
   "require": {
     "php": ">=8.0",
-    "symfony/serializer": "^4.4|^5.0",
+    "symfony/serializer": "^4.4|^5.0|^5.0|^6.0",
```

I'd suggest this, as `>=5.0 <=5.3` is not maintained anymore:

```suggestion
    "symfony/serializer": "^4.4|^5.4|^6.0",
```